.generator/cli.py

```diff
@@ -453,6 +490,34 @@ def _get_libraries_to_prepare_for_release(library_entries: Dict) -> List[dict]:
     ]
 
 
+def _update_global_changelog(source_path: str, output_path: str, libraries: List[dict]):
+    """Updates the versions of libraries in the main CHANGELOG.md.
+
+    Args:
+        source_path(str): Path to the changelog file to read.
+        output_path(str): Path to the changelog file to write.
+        libraries(Dict): Dictionary containing all of the library versions to
+        modify.
+
+    Returns: None
+    """
+
+    def replace_version_in_changelog(content):
+        new_content = content
+        for individual_library in libraries:
```

Comment:
```suggestion
for library in libraries:
```

`individual_library` seems a bit odd. I'd assume library to be a single library. How about the suggestion above? We can also use `all_libraries`, `libraries_to_release`, etc for the latter.